index.html

```diff
@@ -59,9 +59,162 @@ <h1>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>		
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages">
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+		</div>
+	</section>
+	<section class="about-me-section" id="about-me">
+		<div class="container">
+			<h1>About me</h1>
+			<p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+			see and have a project you need coded,
+			don’t hestiate to contact me.</p>
+			<button type="button" >
+				Get My Resume
+			</button>
+		</div>
+		<ul class="about-myself-ul">
+			<li>
+				<div class="skill-category">
+					<div class="category-image">
+						<img src="./img/icon/icon-languages.svg" alt="skill icon">
+					</div>
+					<h2 class="skill-name">Languages</h2>
+					<ul class="skill-list">
+						<li>JavaScript</li>
+						<li>Ruby</li>
+						<li>Html</li>
+						<li>CSS</li>
+					</ul>
+				</div>
```

Comment:
- [ ] Take a look at the points I have highlighted and make the necessary changes to look like the original design, I think you need to add one more framework as per the design on the left

<img width="723" alt="4" src="https://user-images.githubusercontent.com/77843050/212497110-7ab47c25-469b-45c7-af2b-0306b08886df.png">